### PR TITLE
Clean release items

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

File: src/main/java/com/payu/artifactory/tools/releases/ReleasesCleaner.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+        LOGGER.info("Finding items with query: {}", itemsQueryForVersion);
+
+        AQLItemsPath itemsPath = executeQuery(itemsQueryForVersion, AQLItemsPath.class);
+        return itemsPath.getResults().stream().sorted().collect(Collectors.toList());
+    }
+
+    private void deletePath(AQLItemPath path) {
+        LOGGER.info("Delete {}/{}", repo, path.getPath());
+
+        // artifactory.repository(repo).delete(path.getPath());
```
Comment (rrymaszewski) [inline src/main/java/com/payu/artifactory/tools/releases/ReleasesCleaner.java:171]: do we need it?

File: README.md [hunk 2]
```diff
@@ -13,26 +13,45 @@ When a release of `test:test-1.5` is done, all snapshots with maven version lowe
...
+```
+
+where:
+
+| Item                       | Description                                                                                | Defaults |
+|----------------------------|--------------------------------------------------------------------------------------------|----------|
+| repository-name            | remote repository name                                                                     |          |
+| root-path-of-maven-project | Maven groupId exposed as path, that will be used for searching a versions list for removal |          |
+| days                       | artifacts version older than `days` will be scheduled for removal                          | 365      |
+| remain                     | how many version must always be preserved                                                  | 3        |
```
Comment (rrymaszewski) [inline README.md]: should be `versions`

File: README.md [hunk 3]
```diff
@@ -13,26 +13,45 @@ When a release of `test:test-1.5` is done, all snapshots with maven version lowe
...
+| limit                      | how many version should be removed in on execution                                         | 128      |
```
Comment (rrymaszewski) [inline README.md]: should be `versions`

**Review by rrymaszewski — CHANGES_REQUESTED** [review]

File: src/main/java/com/payu/artifactory/tools/releases/ReleasesCleaner.java (hunk 1 above)
Comment (slawekjaranowski) [reply src/main/java/com/payu/artifactory/tools/releases/ReleasesCleaner.java:171]: Yes finally we need it.
In first step I would like to test with a `dryRun` 😄

**Review by rrymaszewski — APPROVED** [review]

**Pull request merged**